Lib/test/test_listcomps.py

```diff
@@ -139,7 +139,7 @@ def test_main(verbose=None):
         import gc
         counts = [None] * 5
         for i in range(len(counts)):
-            support.run_doctest(test_genexps, verbose)
+            support.run_doctest(test_listcomps, verbose)
```

Comment:
This looks like a simple copy bug and fix.  Given that 'test_genexps' is not defined in the file, it seems like a bug that it passes when run by test.regrtest.